client/app/bundles/course/video/submission/containers/VideoPlayer.jsx

```diff
@@ -92,7 +92,7 @@ class VideoPlayer extends React.Component {
           url={this.props.videoUrl}
           playing={isPlayingState(this.props.playerState)}
           volume={this.props.playerVolume}
-          playbackRate={this.props.playbackRate}
+          playbackRate={this.props.playBackRate}
```

Comment:
Is there any reason why it's inconsistent with the variable name?

Reply:
I think it was a porting issue I missed. Previously some other API was used to play the video, and it was `playBackRate` across the board. 

I could do a refactor if that's prefered?